<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>interpolategps</class>
 <widget class="QDialog" name="interpolategps">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>680</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>680</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Interpolate GPS Data</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qmlscripts.qrc">
    <normaloff>:/img/misc/scribeIconFlipped.png</normaloff>:/img/misc/scribeIconFlipped.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #FFFFFF;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_10">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>13</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>8</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QTextEdit" name="txtFile">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777214</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border:none;
border-bottom:solid;
border-width:1px;
border-color:'#E2E2E2';
font: 13px &quot;Segoe UI&quot;;
background-color:'transparent';</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>8</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>13</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnLoad">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>110</width>
           <height>26</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
  border: 1px solid #ADADAD;
  opacity: 1;
  text-align: center;
  font: 13px &quot;Segoe UI&quot;;
  color: #000000;
  background: #E1E1E1;
  margin: 0;
  padding-bottom:4px;
}
QPushButton:hover{
  color: #000000;
  background-color: #E5F1FB;
  border-color: #0078D7;
}
QPushButton:focus{
  #box-shadow: none, 0 0 0 0.2rem rgba(0, 188, 212, 0.5);
}
QPushButto:disabled {
  color: #ffffff;
  background-color: #00bcd4;
  border-color: #00bcd4;
}</string>
         </property>
         <property name="text">
          <string>Load GPS Data</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_11">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>13</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>13</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>1</number>
         </property>
         <item>
          <widget class="QGroupBox" name="grpboxgps">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>250</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QGroupBox{
background-color:  #F7F7F7; 
margin-top: 2ex;
border: 1px solid #D8D8D8; 
font-weight:regular;
font-family:&quot;Segoe UI&quot;;
} 
QGroupBox:title {
subcontrol-position: top left; 
padding: -12px 0px 0px 3px; 
color:#000000;
font-weight:bold;
}</string>
           </property>
           <property name="title">
            <string>Existing GPS Data</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QTableWidget" name="tblgps">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>150</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QTableWidget{
    border: 1px solid &quot;#E2E2E2&quot;;
    opacity: 1;
    background: &quot;#FFFFFF&quot;;
    margin: 0;
	gridline-color: &quot;#E2E2E2&quot;;
}
QHeaderView::section{
    spacing: 10px;
    background-color: &quot;#FFFFFF&quot;;
    color: &quot;#000000&quot;;
    border:none;
    border-bottom: 1px solid &quot;#E2E2E2&quot;;
	border-right: 1px solid &quot;#E2E2E2&quot;;
    margin:0;
	font-size: 13px;
    font-family: &quot;Segoe UI&quot;;
}</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Plain</enum>
              </property>
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="grpboxmissing">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>250</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>250</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QGroupBox{
background-color:  #F7F7F7; 
border: 1px solid #D8D8D8; 
margin-top: 2ex;
font-weight:regular;
font-family:&quot;Segoe UI&quot;;
} 
QGroupBox:title {
subcontrol-position: top left; 
padding: -12px 0px 0px 3px; 
color:black;
}</string>
           </property>
           <property name="title">
            <string>Missing Time</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="leftMargin">
             <number>9</number>
            </property>
            <property name="topMargin">
             <number>9</number>
            </property>
            <property name="rightMargin">
             <number>9</number>
            </property>
            <property name="bottomMargin">
             <number>9</number>
            </property>
            <item>
             <widget class="QTableWidget" name="tblmissing">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QTableWidget{
    border: 1px solid &quot;#E2E2E2&quot;;
    opacity: 1;
    background: &quot;#FFFFFF&quot;;
    margin: 0;
	gridline-color: &quot;#E2E2E2&quot;;
}
QHeaderView::section{
    spacing: 10px;
    background-color: &quot;#FFFFFF&quot;;
    color: &quot;#000000&quot;;
    border:none;
    border-bottom: 1px solid &quot;#E2E2E2&quot;;
	border-right: 1px solid &quot;#E2E2E2&quot;;
    margin:0;
	font-size: 13px;
    font-family: &quot;Segoe UI&quot;;
}</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Plain</enum>
              </property>
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_9">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>13</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>8</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>299</width>
         <height>68</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>68</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background: #EEEEEE;
opacity: 1;
margin: 0;
</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <spacer name="verticalSpacer_7">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>16</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>12</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btnInterpolate">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>26</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>26</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
  border: 1px solid #ADADAD;
  opacity: 1;
  text-align: center;
  font: BOLD 13px &quot;Segoe UI&quot;;
  color: #000000;
  background: #E1E1E1;
  margin: 0;
  padding-bottom:4px;
}
QPushButton:disabled{
  color: #ADADAD;
  background-color: #E1E1E1;
  border-color: #ADADAD;
}
QPushButton:hover{
  color: #000000;
  background-color: #E5F1FB;
  border-color: #0078D7;
}
QPushButton:focus{
  #box-shadow: none, 0 0 0 0.2rem rgba(0, 188, 212, 0.5);
}
</string>
              </property>
              <property name="text">
               <string>Interpolate</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>8</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btnTest">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>26</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>26</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
  border: 1px solid #ADADAD;
  opacity: 1;
  text-align: center;
  font: BOLD 13px &quot;Segoe UI&quot;;
  color: #000000;
  background: #E1E1E1;
  margin: 0;
  padding-bottom:4px;
}
QPushButton:disabled{
  color: #ADADAD;
  background-color: #E1E1E1;
  border-color: #ADADAD;
}
QPushButton:hover{
  color: #000000;
  background-color: #E5F1FB;
  border-color: #0078D7;
}
QPushButton:focus{
  #box-shadow: none, 0 0 0 0.2rem rgba(0, 188, 212, 0.5);
}
</string>
              </property>
              <property name="text">
               <string>Test GPS Data</string>
              </property>
              <property name="autoDefault">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btnclear">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>26</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>26</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
  border: 1px solid #ADADAD;
  opacity: 1;
  text-align: center;
  font: BOLD 13px &quot;Segoe UI&quot;;
  color: #000000;
  background: #E1E1E1;
  margin: 0;
  padding-bottom:4px;
}
QPushButton:disabled{
  color: #ADADAD;
  background-color: #E1E1E1;
  border-color: #ADADAD;
}
QPushButton:hover{
  color: #000000;
  background-color: #E5F1FB;
  border-color: #0078D7;
}
QPushButton:focus{
  #box-shadow: none, 0 0 0 0.2rem rgba(0, 188, 212, 0.5);
}
</string>
              </property>
              <property name="text">
               <string>Clear All</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>8</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btnSave">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>64</width>
                <height>26</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>64</width>
                <height>26</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
  border: 1px solid #ADADAD;
  opacity: 1;
  text-align: center;
  font: BOLD 13px &quot;Segoe UI&quot;;
  color: #000000;
  background: #E1E1E1;
  margin: 0;
  padding-bottom:4px;
}
QPushButton:disabled{
  color: #ADADAD;
  background-color: #E1E1E1;
  border-color: #ADADAD;
}
QPushButton:hover{
  color: #000000;
  background-color: #E5F1FB;
  border-color: #0078D7;
}
QPushButton:focus{
  #box-shadow: none, 0 0 0 0.2rem rgba(0, 188, 212, 0.5);
}
</string>
              </property>
              <property name="text">
               <string>Save</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_7">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>12</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_8">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>16</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>btnLoad</tabstop>
  <tabstop>btnInterpolate</tabstop>
  <tabstop>btnTest</tabstop>
  <tabstop>btnclear</tabstop>
  <tabstop>btnSave</tabstop>
  <tabstop>tblgps</tabstop>
  <tabstop>tblmissing</tabstop>
  <tabstop>txtFile</tabstop>
 </tabstops>
 <resources>
  <include location="qmlscripts.qrc"/>
 </resources>
 <connections/>
</ui>
